cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
@cpennington is it the right place to add this service here ? We ended up adding in here because we've tried to add in previewModuleSystem but it turned out that we override that runtime with this one at some point in item.py. Any thoughts ?